Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -1522,7 +1532,11 @@ std::unique_ptr<RenderStyle> TreeResolver::generatePositionOption(const Position
         .tactics = fallback.tactics
     };
 
-    return resolveAgainInDifferentContext(resolvedStyle, styleable, *resolutionContext.parentStyle, PropertyCascade::normalPropertyTypes(), WTFMove(builderFallback), resolutionContext);
+    auto style = resolveAgainInDifferentContext(resolvedStyle, styleable, *resolutionContext.parentStyle, PropertyCascade::normalPropertyTypes(), WTFMove(builderFallback), resolutionContext);
+    if (style)
+        style->setUsedPositionTryFallbackIndex(fallbackIndex);
```

Comment:
We probably don't need to do this here and avoid propagating the fallbackIndex to these functions.

Reply:
I adopt your change so we only set it when the option is tried.